src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+
+const collectionOfButton = [...thead.rows[0].cells];
+
+for (let i = 0; i < collectionOfButton.length; i++) {
+  collectionOfButton[i].addEventListener('click', () => {
+    [...tbody.rows]
+      .sort((rowA, rowB) => {
+        const salaryA = rowA.lastElementChild.textContent.replace(/[$,]/g, '');
+        const SalaryB = rowB.lastElementChild.textContent.replace(/[$,]/g, '');
```

Comment:
The variable names `salaryA` and `SalaryB` are inconsistent in their casing. It's a good practice to follow a consistent naming convention, such as camelCase, for better readability and maintenance. Consider renaming `SalaryB` to `salaryB`.